java/src/jmri/InstanceManager.java

```diff
@@ -72,9 +78,11 @@
      * @param type The class Object for the item's type. This will be used as
      *             the key to retrieve the object later.
      */
-    static public <T> void store(T item, Class<T> type) {
+    static public <T> void store(T item, @Nonnull Class<T> type) {
```

Comment:
Should item be Nonnull?
We log an error with stack trace if it is null, but then store it anyway. Is that an error?